code/__DEFINES/bloodcult.dm

```diff
@@ -0,0 +1,2 @@
+var/global/blood_target = null

```

Comment:
Well if its not a define, and apparently doesnt need to be, where should I put them?